### PR TITLE
CASMTRIAGE-4334 update upgrade docs to check if csm-rbd-mount already exists before creating it

### PR DESCRIPTION
**Pull request by leliasen-hpe**
# Description

Docs change. Update upgrade docs to check if csm-rbd-mount already exists before creating it.

# Checklist Before Merging

- [x] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [x] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [x] My commits or Pull-Request Title contain my JIRA information, or I do not have a JIRA.

[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

File: upgrade/Stage_0_Prerequisites.md [hunk 1]
```diff
@@ -84,10 +84,23 @@ after a break, always be sure that a typescript is running before proceeding.
 
    1. Create and map the `rbd` device.
 
-      **IMPORTANT:** This mounts the `rbd` device at `/etc/cray/upgrade/csm` on `ncn-m001`. This mount is available to stage content for the install/upgrade process.
+      **IMPORTANT:** This mounts the `rbd` device at `/etc/cray/upgrade/csm` on `ncn-m001`. This mount is available to stage content for the install/upgrade process. First, check if this device already exists.
 
       ```bash
       source /opt/cray/csm/scripts/csm_rbd_tool/bin/activate
+      python /usr/share/doc/csm/scripts/csm_rbd_tool.py --status
```
Comment (cdelatte-hpe) [inline upgrade/Stage_0_Prerequisites.md:91]: we should not need to specify python to run the script.

**Review by bklei — APPROVED** [review]

File: upgrade/Stage_0_Prerequisites.md [hunk 2]
```diff
@@ -84,10 +84,23 @@ after a break, always be sure that a typescript is running before proceeding.
...
+      ```
+
+      If the `rbd` device already exists and is mounted, it can be moved to the desired node, if not already mounted there.
+
+      ```bash
+      python /usr/share/doc/csm/scripts/csm_rbd_tool.py --rbd_action move --target_host ncn-m001
```
Comment (cdelatte-hpe) [inline upgrade/Stage_0_Prerequisites.md:97]: we should not need to specify python to run the script.

File: upgrade/Stage_0_Prerequisites.md [hunk 3]
```diff
@@ -84,10 +84,23 @@ after a break, always be sure that a typescript is running before proceeding.
...
+      deactivate
+      ```
+
+      If the `rbd` device does not exists, create it.
+
+      ```bash
       python /usr/share/doc/csm/scripts/csm_rbd_tool.py --pool_action create --rbd_action create --target_host ncn-m001
```
Comment (cdelatte-hpe) [inline upgrade/Stage_0_Prerequisites.md:104]: we should not need to specify python to run the script.

**Review by cdelatte-hpe — CHANGES_REQUESTED** [review]
also provide some output so people know what they are expecting

File: upgrade/Stage_0_Prerequisites.md (hunk 1 above)
Comment (cdelatte-hpe) [inline upgrade/Stage_0_Prerequisites.md:91]:
```suggestion
      /usr/share/doc/csm/scripts/csm_rbd_tool.py --status
```

File: upgrade/Stage_0_Prerequisites.md (hunk 2 above)
Comment (cdelatte-hpe) [inline upgrade/Stage_0_Prerequisites.md:97]:
```suggestion
      /usr/share/doc/csm/scripts/csm_rbd_tool.py --rbd_action move --target_host ncn-m001
```

File: upgrade/Stage_0_Prerequisites.md (hunk 3 above)
Comment (cdelatte-hpe) [inline upgrade/Stage_0_Prerequisites.md:104]:
```suggestion
      /usr/share/doc/csm/scripts/csm_rbd_tool.py --pool_action create --rbd_action create --target_host ncn-m001
```

File: upgrade/Stage_0_Prerequisites.md (hunk 1 above)
Comment (leliasen-hpe) [reply upgrade/Stage_0_Prerequisites.md]: removed python

**Review by cdelatte-hpe — APPROVED** [review]

**Review by rustydb — APPROVED** [review]

**Review by gbaker-hpe — APPROVED** [review]

**Pull request merged**